src/libraries/System.Security.Cryptography/tests/HKDFTests.cs

```diff
@@ -15,25 +15,25 @@ public abstract class HKDFTests
         protected abstract byte[] DeriveKey(HashAlgorithmName hash, byte[] ikm, int outputLength, byte[] salt, byte[] info);
 
         [Theory]
-        [MemberData(nameof(GetRfc5869TestCases))]
-        public void Rfc5869ExtractTests(Rfc5869TestCase test)
+        [MemberData(nameof(GetHkdfTestCases))]
+        public void Rfc5869ExtractTests(HkdfTestCase test)
```

Comment:
Should the `Rfc5869` prefix be removed from all of these methods (to match renaming the type and property)?